unstructured_inference/inference/layout.py

```diff
@@ -484,6 +470,8 @@ def process_file_with_model(
     fixed_layouts: Optional[List[Optional[List[TextRegion]]]] = None,
     extract_tables: bool = False,
     pdf_image_dpi: int = 200,
+    extract_images_in_pdf: bool = False,
+    image_output_dir_path: Optional[str] = None,
```

Comment:
what's the guide line here? When do we explicit set and when do we implicitly trust a kwarg does something down the call stack? And why are those two explicit here? It doesn't seem those two parameters are modified in this function and they are just passing through?

Reply:
These are explicit since we don't want them passed down to `get_model`, but we do want them passed down to `.from_file`